homeassistant/components/cast/media_player.py

```diff
@@ -423,6 +575,14 @@ def async_cast_removed(discover: ChromecastInfo):
             async_cast_removed)
         self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, async_stop)
         self.hass.async_create_task(self.async_set_cast_info(self._cast_info))
+        for info in self.hass.data[KNOWN_CHROMECAST_INFO_KEY]:
+            if _is_our_dynamic_group(self._cast_info, info):
```

Comment:
I think that `is_our` is a weird naming. I would suggest `member_of` ?

Reply:
A dynamic group is formed when a group is streaming, and one of the group members is being told to stream something else.
We're not looking for _members_ of the dynamic group here, we want the the real group to get its state from the dynamic group until it's killed (happen as soon as the session stops).

Reply:
`is_dynamic_group_by_member_of_my_group` 😆

Reply:
Renamed to `_is_matching_dynamic_group`, maybe better?